### DISCUSSION TITLE
Are the E-field maps created by SimNIBS adjusted to the average percentage of the maximal stimulator output?

### DISCUSSION THREAD

**Discussion by gcotovio** [simnibs/simnibs#144] [8. Simulation workflows]
Dear developers,

Thank you for your great work. I apologize in advance for my ignorance in this field.

I was wondering if the E-field maps (Output Nifti files) created by SimNIBS adjusted to the average percentage of the maximal stimulator output? What is the default stimulator output that is used? How can we change it appropriately, for example to a specfic % of a subject motor threshold?

Thank you again.

**Comment by oulap** [comment]
Simnibs uses the dI/dt value for scaling the e-fields for TMS. Please see the discussion in our FAQ:

https://simnibs.github.io/simnibs/build/html/faq.html

tl;dr
You can scale the maximum dI/dt of your specific coil and stimulator setup with the percentage of maximum stimulator output that was used to obtain the dI/dt that you should input into simnibs simulations.
